### ISSUE TITLE
Permalinks should be native `<a>` links

### ISSUE THREAD

**Issue by davilima6** [gitterHQ/gitter#950]
It should be possible to right click and have a normal "Copy link location" in permalinks.

**Comment by abhisekp** [comment]
Or better to just auto copy on click on timestamp.

**Comment by davilima6** [comment]
Both! But agree with @abhisekp - nice touch

**Comment by MadLittleMods** [comment]
@davilima6 Your last suggestion ties more into using semantic elements all around, see https://github.com/gitterHQ/gitter/issues/538

**Comment by MarcoZehe** [comment]
What's the plan on going forward with this?

**Comment by MadLittleMods** [comment]
Hey @MarcoZehe, coincidence that I started on this yesterday and should be done soon and in the next release.

**Comment by MadLittleMods** [comment]
Permalinks are now native `<a>` links. This will be available when we cut the next release.

**Comment by davilima6** [comment]
Thx!
